pyramid_zipkin/request_helper.py

```diff
@@ -79,18 +80,16 @@ def should_not_sample_route(request):
     return (route_info and route_info.name in blacklisted_routes)
 
 
-def should_sample_as_per_zipkin_tracing_percent(tracing_percent, req_id):
+def should_sample_as_per_zipkin_tracing_percent(tracing_percent):
     """Calculate whether the request should be traced as per tracing percent.
 
     :param tracing_percent: value between 0.0 to 100.0
     :type tracing_percent: float
-    :param req_id: unique request id of the request
     :returns: boolean whether current request should be sampled.
     """
     if tracing_percent == 0.0:  # Prevent the ZeroDivision
```

Comment:
This no longer prevents ZeroDivision. You can get rid of this if you want, or you can just remove the comment. It's probably _marginally_ faster if tracing_percent == 0.0.